force-app/main/default/lwc/givingSummary/givingSummary.html

```diff
@@ -0,0 +1,35 @@
+<template>
```

Comment:
What happen if the user does not have access to the 3 fields on the Contact. Some sort of error handling is required in case anything goes wrong.

Reply:
agreed